tools/benchmark/binsize/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
```

Comment:
You should the the license header for all new *.toml files. It's like the `//` headers for your Rust source, [but instead with `#`](https://github.com/unicode-org/icu4x/blob/main/tools/scripts/tidy.toml#L31-L41).

If you just want to run the license header check locally instead of running on CI, you can run `cargo make license-header-check`.  If you want to run the small set of small checks like Rust source formatter, Rust linter, license header, readme auto-generation, then you can run `cargo tidy`. And to run a larger subset of CI jobs locally (incl. unit tests, etc.), run `cargo quick`.